@here/harp-mapview/lib/DataSource.ts

```diff
@@ -63,6 +63,7 @@ export interface DataSourceOptions {
     maxDisplayLevel?: number;
     /**
      * Storage level offset applied to this `DataSource`.
+     * @default 0
```

Comment:
Isn't the default "datasource dependent"? e.g. in HereWebTileDataSource.ts the default seem to be -1.

Reply:
Yes, I refer to the DataSource class itself, which defaults to 0. But I can remove if you prefer?

Reply:
I have removed it.